modules/features/falcon_feature_user/config/install/user.role.anonymous.yml

```diff
@@ -0,0 +1,12 @@
+langcode: en
+status: true
+dependencies: {  }
+id: anonymous
+label: 'Anonymous user'
+weight: 0
+is_admin: false
+permissions:
+  - 'access checkout'
+  - 'access content'
+  - 'view commerce_product'
```

Comment:
Do we need anonymous and authenticated roles to have access to products in decoupled architecture?

Reply:
Not sure about it; removed for now.